src/Sylius/Behat/Page/Shop/Order/ShowPage.php

```diff
@@ -59,6 +59,19 @@ public function getAmountOfItems(): int
         return count($paymentItems);
     }
 
+    public function getChosenPaymentMethod(): string
+    {
+        $paymentMethodItems = $this->getDocument()->findAll('css', '[data-test-payment-item]');
+
+        foreach ($paymentMethodItems as $method) {
+            if ($method->find('css', '[data-test-payment-method-select]')->hasAttribute('checked')) {
+                return $method->find('css', 'a')->getText();
```

Comment:
Shouldn't we provide a custom test data attribute for a link as well?

Reply:
here i am iterating over the shipping method and IMO my approach is more readable, this link is also a label form shipping method.